app/(tabs)/trends.tsx

```
@@ -24,36 +25,91 @@ export default function TrendsScreen() {
+    [categories, ouraConnected, whoopConnected]
   );
 
   const currentCategory = availableCategories.find((category) => category.id === selectedFilter);
 
+  useEffect(() => {
+    if (!autoSelectedRef.current.oura && ouraConnected) {
+      const ouraCategory = categories.find((category) => category.id === 'Oura');
+      if (ouraCategory && !ouraCategory.charts.some((chart) => selectedCharts.has(chart.id))) {
+        setSelectedCharts((prev) => {
+          const next = new Set(prev);
+          ouraCategory.charts.forEach((chart) => next.add(chart.id));
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)